### DISCUSSION TITLE
Google rolls back decision to kill third-party cookies in Chrome ??

### DISCUSSION THREAD

**Discussion by rhsl** [privacysandbox/privacy-sandbox-dev-support#407] [Phasing out third-party cookies]
Hi
According to the news below, Google is revoking its 3rd party cookie block policy.
https://www.bleepingcomputer.com/news/security/google-rolls-back-decision-to-kill-third-party-cookies-in-chrome/

Are 3rd party cookies actually maintained?
Is there an official announcement about the revocation?

**Comment by samdutton** [comment]
Hi @rhsl 

[A new path for Privacy Sandbox on the web](https://privacysandbox.com/news/privacy-sandbox-update/) explains Chrome's plans.
